src/main/java/mate/academy/spring/dto/response/UserResponseDto.java

```diff
@@ -1,8 +1,12 @@
 package mate.academy.spring.dto.response;
 
+import java.util.Set;
+import mate.academy.spring.model.Role;
+
 public class UserResponseDto {
     private Long id;
     private String email;
+    private Set<Role> roles;
```

Comment:
dont return entities in dto
```suggestion
    private Set<String> roles;
```